bin/rad-diff

```diff
@@ -153,16 +153,46 @@
   (fn [state]
     (drop 1 (show state))))
 
+(def cells
+  {:state       10
+   :number      10
+   :title       50
+   :author      20
+   :modified-at 20})
+
+(def wrap-cell
```

Comment:
Can you move 
a) abstract a `pad-to` from this (that takes a str and a length and fills it with spaces or cuts it out)
b) move it to `prelude/strings` 
c) and write a couple of tests?

Reply:
Due to some weird build failures I had to revert the changes extracting the function. It has to be done at a later time, when we know where the problems came from. (For further investigation check out https://github.com/oscoin/radicle/pull/458)